src/common/CommonUtils.ts

```diff
@@ -234,6 +234,24 @@ export class CommonUtils {
         );
     }
 
+    /**
+     * Launches the desktop browser and navigates to the provided URL.
+     *
+     * @returns A Promise that launches the desktop browser and navigates to the provided URL.
+     */
+    public static async launchUrlInDesktopBrowser(url: string): Promise<void> {
```

Comment:
This is used in other repos but I thought to bring it here since it is a useful utility function, and instead call into it from other repos.